server/routes/liability.js

```diff
@@ -1,10 +1,55 @@
 const router = require('express').Router();
-const { Liability } = require('../db');
+const { Liability, Device, User } = require('../db').models;
 
-function check(req, res) {
-  res.send('It works!!');
+const DEFAULT_LIMIT = 30;
+
+function create(req, res, next) {
+  return Liability.create(req.body)
+    .then(liability => res.send(liability))
+    .catch(err => next(err));
+}
+
+function get(req, res, next) {
+  const where = { id: req.params.id };
+  const include = [{ model: Device }, { model: User }];
+
+  return Liability.find({ where, include })
+    .then(liability => res.send(liability))
+    .catch(err => next(err));
+}
+
+function list(req, res, next) {
+  const limit = Number(req.query.limit) || DEFAULT_LIMIT;
+  const offset = Number(req.query.offset) || 0;
+  const where = req.query;
+  const include = [{ model: Device }, { model: User }];
+
+  return Liability.findAll({ where, limit, offset, include })
```

Comment:
Be consistent about ordering because `where` and `include` are used to determine data while `limit` and `offset` determine selection window. :bowtie: